services/web/server/src/simcore_service_webserver/payments/_methods_api.py

```diff
@@ -65,7 +65,9 @@ def _to_api_model(
 
 
 #
-# Payment-methods
+# Payment Methods:
+# - Resources is called `wallet_payment_method`
```

Comment:
Q: is this comment useful? cause for me it makes no sense.

Reply:
I used them to give an idea on the naming and guide the reader but obviously did not work :-) I will remove them